instaloader/instaloadercontext.py

```diff
@@ -528,6 +528,25 @@ def get_and_write_raw(self, url: str, filename: str) -> None:
         :raises ConnectionException: When download repeatedly failed."""
         self.write_raw(self.get_raw(url), filename)
 
+    def head(self, url: str, allow_redirects: Optional[bool] = None) -> requests.Response:
```

Comment:
```suggestion
    def head(self, url: str, allow_redirects: bool = False) -> requests.Response:
```

Otherwise, we would pass `allow_redirects=None` to `anonymous_session.head()` if no `allow_redirects` parameter is specified, which looks wrong to me.

Reply:
Thanks! I was confused by the requests documentation ad the time, and thought there was a difference between None and False. Turns out it just has different default value (`False`) for `.head()`.